src/main/java/nl/tudelft/dnainator/ui/DNAinator.java

```diff
@@ -24,20 +26,31 @@
 	private static final int MIN_WIDTH = 300;
 	private static final int MIN_HEIGHT = 150;
 
+	private Stage stage;
```

Comment:
I agree there is no urgent need for a separate method, but IMO it is better to have one for method length control (aim is 12 lines at max) and method responsibility.